Python/sysmodule.c

```diff
@@ -2438,6 +2438,7 @@ static PyStructSequence_Field flags_fields[] = {
     {"quiet",                   "-q"},
     {"hash_randomization",      "-R"},
     {"isolated",                "-I"},
+    {"use_peg",                 "-p"},
```

Comment:
I'd suggest using `"-p new"` for the docstring (following the example for the `-X` flags below, or even `"-p old or -p new"` (after `optimize`).